NitroxModel/DataStructures/GameLogic/Entity.cs

```diff
@@ -9,7 +10,7 @@ namespace NitroxModel.DataStructures.GameLogic
     [ProtoContract]
     public class Entity
```

Comment:
I think in the future (probably not this PR).  We'll want to break up into different types of entities:

`DroppedEntity` - has serializedGameObject and only basic elements
`LocallyPositionedEntity`
`GloballyPositionedEntity`